apps/nextra/pages/en/build/guides/build-e2e-dapp/4-submit-data-to-chain.mdx

```diff
@@ -49,7 +46,7 @@ const addNewList = async () => {
     // sign and submit transaction to chain
     const response = await signAndSubmitTransaction(transaction);
     // wait for transaction
-    await aptos.waitForTransaction({transactionHash:response.hash});
+    await aptosClient().waitForTransaction({transactionHash:response.hash});
```

Comment:
New SDK syntax?

Reply:
just a template util function to reuse the same Aptos client